src/main/java/org/generationcp/middleware/manager/ontology/OntologyVariableDataManagerImpl.java

```diff
@@ -803,7 +823,9 @@ public void deleteVariable(final Integer variableId) {
 		this.checkTermIsVariable(term);
 
 		// check usage
-		final Boolean isUsed = this.isVariableUsedInStudy(variableId);
+		final Boolean isUsed = this.isVariableUsedInStudy(variableId) || //
+			this.isVariableUsedInGermplasm(variableId) || //
+			this.isVariableUsedInBreedingMethods(variableId);
```

Comment:
we should have a central method if variable is used anywhere (studies, germplasm, breeding methods, listdataprops). In this specific instance since the code is duplicated, it is missing the check in listdataprops

Reply:
done.